VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/vho-global-styles.scss

```diff
@@ -0,0 +1,13 @@
+//MAKE UI FILL THE PAGE WIDTH
+.govuk-width-container {
+  max-width: none;
+  //min-height: 1600px;
+  //padding:0;
+  margin: 0;
```

Comment:
Sorry mate, more commenting out here. I missed it the first time

Reply:
removed the class, it wasn't meant to have been committed